pypesto/visualize/sampling.py

```diff
@@ -61,7 +61,10 @@ def sampling_fval_trace(
               's': 10}
     if full_trace:
         kwargs['hue'] = "converged"
-        kwargs['palette'] = ["#868686", "#477ccd"]
+        if len(params_fval[kwargs['hue']].unique()) == 1:
+            kwargs['palette'] = ["#477ccd"]
+        elif len(params_fval[kwargs['hue']].unique()) == 2:
```

Comment:
it cannot be more than 2?

Reply:
good question, I thought the converged status can only be '0' or '1', but I'm not sure about that, are there other possibilities for the status 'converged'?

Reply:
converged is a binary indicator for whether this is still burn-in or the chain after burn-in, right? yes, then it should be one or 2.